Patterns/Italian/Italian-DateTime.yaml

```diff
@@ -610,11 +657,6 @@ UnitValueMap: !dictionary
     secondo: 1
     secs: 1
     sec: 1
-# TODO: modify below regex according to the counterpart in English
-SpecialYearPrefixesMap: !dictionary
```

Comment:
Aren't there any counterparts of "fiscal year" and "school year" in Italian?

Reply:
updated